int32 motor_num

#feedback gain
float32[] pos_p_gain_throttle
float32[] pos_i_gain_throttle
float32[] pos_d_gain_throttle

float32[] pos_p_gain_yaw
float32[] pos_i_gain_yaw
float32[] pos_d_gain_yaw

#feedforward gain
float32[] roll_vec
float32[] pitch_vec
float32[] yaw_vec
